scaladoc/src/dotty/tools/scaladoc/ScaladocSettings.scala

```diff
@@ -43,7 +43,7 @@ class ScaladocSettings extends SettingGroup with AllScalaSettings:
 
   val socialLinks: Setting[List[String]] =
     MultiStringSetting("-social-links", "social-links",
-      "Links to social sites. '[github|twitter|gitter|discord]::link' syntax is used.")
+      "Links to social sites. '[github|twitter|gitter|discord]::link' or 'custom::link::light_icon_file_name[::dark_icon_file_name]' syntax is used. For custom links, the icons must be present in '_assets/images/'")
```

Comment:
Also here we have `custom` and when we match on this we match on `custom`. Should we instead allow them to use the name of their choice? For example let's say I want to use a link to my mastodon account, I may want to do `mastodon::link::light::dark`.

Reply:
Ho Indeed, this can be useful, and I'm going to try to do it. For example, if I want to put several custom icons, I have this kind of settings :
```social-links:custom::https://www.epfl.com::epfl.png,custom::scala.com::scala_logo.png```
So it can be useful to have:
```social-links:epfl::https://www.epfl.com::epfl.png,scala::scala.com::scala_logo.png```

Reply:
Hello Chris! In my last commit, I added a way for the user to use a name for their custom icon using a regex. It's a first idea and I'm very interested if you have any corrections, other ideas or suggestions.